@xen-orchestra/lite/src/components/RemoteConsole.vue

```diff
@@ -21,7 +21,8 @@ const props = defineProps<{
 const N_TOTAL_TRIES = 8
 const FIBONACCI_MS_ARRAY: number[] = Array.from(fibonacci().toMs().take(N_TOTAL_TRIES))
 
-const vmConsoleContainer = ref<HTMLDivElement>()
+const vmConsoleContainer = useTemplateRef('vmConsoleContainer')
```

Comment:
you can keep the type
```suggestion
const vmConsoleContainer = useTemplateRef<HTMLDivElement>('vmConsoleContainer')
```